source/extensions/filters/http/jwt_authn/filter.h

```diff
@@ -47,7 +47,7 @@ class Filter : public Http::StreamDecoderFilter,
   // Filter config object.
   FilterConfigSharedPtr config_;
   // Verify context for current.
-  VerifyContextPtr context_;
+  VerifyContextSharedPtr context_{};
```

Comment:
not need for {} for object with default constructor, such as shared_ptr.  Only the primary types need it